qiskit_experiments/database_service/db_experiment_data.py

```diff
@@ -99,12 +99,14 @@ def __init__(
         experiment_type: Optional[str] = "Unknown",
         backend: Optional[Union[Backend, BaseBackend]] = None,
         experiment_id: Optional[str] = None,
+        parent_id: Optional[str] = None,
         tags: Optional[List[str]] = None,
         job_ids: Optional[List[str]] = None,
         share_level: Optional[str] = None,
         metadata: Optional[Dict] = None,
         figure_names: Optional[List[str]] = None,
         notes: Optional[str] = None,
+        verbose: Optional[bool] = True,
```

Comment:
Making this a class attribute we can remove this kwarg
```suggestion
        verbose: Optional[bool] = True,
```